### PR TITLE
[ALFA-996] Add common nats helper functions

### PR DESCRIPTION
**Pull request by sm-amykhalkiv**

### CODE REVIEW

File: nats/common.go
Comment (antonsapyanov) [inline nats/common.go:7]: @sm-amykhalkiv do you need to export these constants? Maybe the constants could be replaced with the literals then?
```go
func getRegionFromHeader(m *nats.Msg) string {
	return getHeaderValue(m, "region")
}
```

File: nats/common.go
Comment (antonsapyanov) [inline nats/common.go:1]: `nats/headers.go`?

File: nats/common.go
Comment (sm-amykhalkiv) [reply nats/common.go:7]: @antonsapyanov I don't right now to export them, jut to use helper functions

File: nats/common.go
Comment (sm-amykhalkiv) [reply nats/common.go:1]: @sm-amykhalkiv Looks more suitable

**Review by jhaynie — APPROVED** [review]

**Comment by jhaynie** [comment]
@sm-amykhalkiv i will merge my other PR once this merges and do an official release here shortly to combine both PRs

**Pull request merged**
